.golangci.yml

```diff
@@ -0,0 +1,41 @@
+run:
+  deadline: 10m
+
+linters:
+  enable:
+    - contextcheck
+    - errcheck
+    - gocritic
+    - gofmt
+    - goimports
+    - gosec
+    - gosimple
+    - govet
+    - ineffassign
+    - misspell
+    - staticcheck
+    - unconvert
+  disable: [unused]
```

Comment:
Why do we disable this here AND add the exception.

Also, although we only have one element, lets use: - unused instead for simpler diffs.